src/Catty/Helpers/Util/Util.swift

```diff
@@ -132,4 +187,18 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         let statusBarSize = UIApplication.shared.statusBarFrame.size
         return min(statusBarSize.width, statusBarSize.height)
     }
+
+    class func sound(withName objectName: String?, forObject object: SpriteObject?) -> Sound? {
```

Comment:
See my previous comment (again `objectName` is confusing since it this parameter is used in a different context).

Reply:
I've changed the method signature following your suggestions, and yes you are right `objectName` was referring to the name of the sound in this case and this really was confusing.